script/libuv.py

```diff
@@ -30,7 +49,7 @@ def download_libuv():
   # version number in this script changes.
   if os.path.isdir(LIB_UV_DIR):
     print("Cleaning output directory...")
-    shutil.rmtree(LIB_UV_DIR)
+    shutil.rmtree(LIB_UV_DIR, onerror=rmtree_onerror)
```

Comment:
Oh, Windows. Always gotta make things difficult!

Instead of this, how do you feel about just detecting the OS and on Windows calling:

```python
subprocess.check_call(['cmd', '/c', 'rd', '/s', '/q', path])
```

instead of `shutil.rmtree()`?

That way, Linux and Mac are left alone.